sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultBackupOperation.cs

```diff
@@ -26,7 +26,7 @@ public class KeyVaultBackupOperation : Operation<KeyVaultBackupResult>
         private RequestFailedException _requestFailedException;
 
         /// <summary>
-        /// Creates an instance of a BackupOperation from a previously started operation. <see cref="UpdateStatus(CancellationToken)"/>, <see cref="UpdateStatusAsync(CancellationToken)"/>,
+        /// Creates an instance of a KeyVaultBackupOperation from a previously started operation. <see cref="UpdateStatus(CancellationToken)"/>, <see cref="UpdateStatusAsync(CancellationToken)"/>,
```

Comment:
The client wraps the type name in `<see cref="..." />` but we're not doing that here.   Should we treat them consistently?

Reply:
IMO, we should, but the generator doesn't define refs now and I think it should. So this would be inconsistent with any generated methods, of which there are more for the admin package. Hence why I made this particular decision here. At least with the class, there's only the one definition of said class.

/cc @m-nash if it's worth changing now, given to be consistent we'd have to regenerate and change any non-ref to a ref in hand-authored code.

Reply:
Fair enough; I agree that it doesn't add enough value for the effort, currently.   It would be nice if the generator could emit this.

Reply:
I opened https://github.com/Azure/autorest.csharp/issues/2992 to track.